app/pb_kits/playbook/pb_circle_icon_button/_circle_icon_button.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= content_tag(:div,
+  id: object.id,
+  data: object.data,
+  class: object.classname) do %>
+
+  <%= pb_rails("button", props: {variant: object.variant, disabled: object.disabled, dark: object.dark}) do %>
+    <%= pb_rails("icon", props: {icon: object.icon, fixed_width: true, dark: object.dark}) %>
```

Comment:
Shouldn't this be using [Icon Circle](https://playbook.powerapp.cloud/kits/icon_circle) instead of `Icon` directly?